input/pagecontent/volume-1.md

```diff
@@ -154,21 +154,21 @@ A Report Repository may modify how the embedded rendered report can be accessed,
 
 A Report Reader displays reports, including [observation imaging context](#152415-observation-imaging-context-in-report) referenced in the report. Based on the referenced `ImagingSelection` resources defined in each observation imaging context, the Report Reader generates corresponding hyperlinks when the report is displayed to the user. When users click on the hyperlinks, the Report Reader presents the initial view of the referenced images to the user, such that the user can interact with the images (e.g., windowing, zooming, panning, toggle annotations, etc.).
 
-##### 1.52.1.1.3.1 Display Requirements
```

Comment:
Good catch. In this case, I think there are other section headings that are still using 1.52 instead of 1:52.